micro_ros_setup/config/nuttx/olimex-stm32-e407/configure.sh

```diff
@@ -0,0 +1,21 @@
+NUTTX_DIR=$FW_TARGETDIR/NuttX
+MCU_WS_DIR=$FW_TARGETDIR/mcu_ws
+
+apt install -y gperf openocd automake git
+
+if [ ! -d ~/tools/kconfig-frontends  ]; then
+    git clone https://bitbucket.org/nuttx/tools.git ~/tools
+
+    pushd ~/tools/kconfig-frontends >/dev/null
+    ./configure --enable-mconf --disable-nconf --disable-gconf --disable-qconf 
+    LD_RUN_PATH=/usr/local/lib && make && make install && ldconfig
+    popd >/dev/null
+fi
+
+
+pushd $NUTTX_DIR >/dev/null
+make distclean
+tools/configure.sh configs/olimex-stm32-e407/uros
```

Comment:
I'm worried about hardcoding any particular configuration. That said, having the user do the configuration manually isn't ideal.

How about we make the configuration to use a command-line argument?

Reply:
Do I understand you correctly, that in `micro_ros_setup/config/nuttx/`, there would be a configure.sh which accepts arguments?

That sounds like a good solution to me.